numba/unicode.py

```diff
@@ -810,3 +813,13 @@ def iternext_unicode(context, builder, sig, args, result):
         # bump index for next cycle
         nindex = cgutils.increment_index(builder, index)
         builder.store(nindex, iterobj.index)
+
+
+ll.add_symbol('str2int', string_conversion_ext.str2int)
```

Comment:
Please reuse the machinery with `_helperlib` to expose C functions.  The manual registration of `str2int` here will not work with ahead-of-time compiled code.